<launch>
    <arg name="image_topic" default="/team220/camera/rgb/compressed"/>
    <!-- <arg name="weight_path" default="$(find mobilenet)/model/model.h5"/> -->
    <arg name="weight_path" default="$(find mobilenet)/model/model_dira.h5"/>


    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
		<param name="port" value="9090"/>
	</include>

    <node name="file_server" pkg="file_server" type="file_server" output="screen"/>

    <!-- <node name="rqt_graph" pkg="rqt_graph" type="rqt_graph" output="screen" /> -->

    <node pkg="mobilenet" type="mobilenet_node.py" name="mobilenet_node" output="screen">
        <param name="image_topic" value="$(arg image_topic)" />
        <param name="weight_path" value="$(arg weight_path)" />
    </node>
</launch>
